docs/understanding-airbyte/typing-deduping.md

```diff
@@ -0,0 +1,68 @@
+# Typing and Deduping
+
+This page refers to new functionality currently available in **early access**. Typing and deduping will become the new default method of transforming datasets after they've been replicated to your destination. This functionality is going live with [Destinations V2](https://github.com/airbytehq/airbyte/issues/26028), which is now in early access for BigQuery.
+
+You will eventually be required to upgrade your connections to use the new destination versions.  We are building tools for you to copy your connector’s configuration to a new version to make testing new destinations easier. These will be available in the next few weeks.
+
+## What is Destinations V2?
+
+At launch, Airbyte Destinations V2 will provide:
+* One-to-one table mapping: Data in one stream will always be mapped to one table in your data warehouse. No more sub-tables.
+* Improved error handling with `_airbyte_meta`: Airbyte will now populate typing errors in the `_airbyte_meta` column instead of failing your sync. You can query these results to audit misformatted or unexpected data.
```

Comment:
```suggestion
* Improved per-row error handling with `_airbyte_meta`: Airbyte will now populate typing errors in the `_airbyte_meta` column instead of failing your sync. You can query these results to audit misformatted or unexpected data.
```